### PR TITLE
Show more/show less button on comment

### LINKED ISSUE #2564: Introduce character limits on comments
We need to update the designs so that:
1. There's a char limit for displaying comments and the "show more" button,
2. There's a chat limit for posting a new comment.

### PR DESCRIPTION
**Pull request by drillprop**
Fix #2564 
Designs: 
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=3677%3A265878

Be aware that the look of the show more button differs from the design. This won't be an issue once https://github.com/Joystream/atlas/issues/2463 is done

I didn't handle [exceeding characters limit story](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/?node-id=3677%3A254185). I'll create a separate issue just for that.

### CODE REVIEW

File: packages/atlas/src/components/_comments/CommentBody/CommentBody.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useRef, useState } from 'react'
+
+import { SvgActionChevronB, SvgActionChevronT } from '@/components/_icons'
+
+import { ExpandButton, StyledCommentText } from './CommentBody.styles'
+
+export const CommentBody = ({ children }: { children?: string }) => {
```
Comment (rafalpawlow) [inline packages/atlas/src/components/_comments/CommentBody/CommentBody.tsx:7]: Can't we just add proper type here?
```suggestion
export const CommentBody: React.FC = ({ children }) => {
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
1. there is wrong spacing between author and deleted comment. Can we fix this as part of this PR? We have `8px` but in figma it's `12px`
Zrzut ekranu 2022-05-10 o 13 37 37

2. I've noticed that Show more button has wrong color but I assume it will be fixed here https://github.com/Joystream/atlas/pull/2637 .

File: packages/atlas/src/components/_comments/CommentBody/CommentBody.tsx (hunk 1 above)
Comment (drillprop) [reply packages/atlas/src/components/_comments/CommentBody/CommentBody.tsx:7]: I want to be able to pass **only** string here, not React.ReactNode, that's why I wrote this.

**Comment by drillprop** [comment]
> 1. there is wrong spacing between author and deleted comment. Can we fix this as part of this PR? We have `8px` but in figma it's `12px`

Fixed

> 2. I've noticed that Show more button has wrong color but I assume it will be fixed here https://github.com/Joystream/atlas/pull/2637 . 

correct

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**